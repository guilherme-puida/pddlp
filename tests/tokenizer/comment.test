>= 10 ; this is a comment
;; line with only comment
<
