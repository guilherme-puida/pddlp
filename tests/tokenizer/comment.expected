[01:01] PDDLP_TOKEN_GTE >=
[01:04] PDDLP_TOKEN_NUMBER 10
[03:01] PDDLP_TOKEN_LT <
EOF